### PR TITLE
Format Change.

### PR DESCRIPTION
**Pull request by dataswifty**
Scalafmt

### CODE REVIEW

File: .scalafmt.conf [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+# Exclude things in here
+project.excludeFilters = [.scalafmt.conf, "FunctionExecutionDispatcher.scala"]
+# Only format files tracked by git.
+project.git = true
+
+# change import a.b.{c,d} => import a.b.{ c, d }
+spaces.inImportCurlyBraces = true
+
+# vertically align function arguments with an arity of 2 or more
```
Comment (augustinas) [inline .scalafmt.conf:12]: I suggest updating to 3 or more. Opinions?

Comment (dataswifty) [reply .scalafmt.conf:12]: I want all of them split for consistency sake, but I want others to weigh-in.

Comment (augustinas) [reply .scalafmt.conf:12]: yup, my motivation is primarily driven by maintenance of a reasonable line length and on average I would say that 2 parameters are still ok for a single line. 3 makes the line too long usually.

**Comment by augustinas** [comment]
Another rule to consider from another project review:
```scala
for {
      a <- Tables.ApplicationDeveloper
      if developerId === a.id
      v <- Tables.ApplicationVersion
    } yield ...
```
Everyone seem to agree that this pattern with `if` filter in between the lines is hard to read. Can we change that? I'm happy to look into it.

**Comment by dataswifty** [comment]
> Another rule to consider from another project review:
> 
> ```scala
> for {
>       a <- Tables.ApplicationDeveloper
>       if developerId === a.id
>       v <- Tables.ApplicationVersion
>     } yield ...
> ```
> 
> Everyone seem to agree that this pattern with `if` filter in between the lines is hard to read. Can we change that? I'm happy to look into it.

Can we look into that later?

File: project/plugins.sbt [hunk 2]
```diff
@@ -1,7 +1,16 @@
 logLevel := Level.Warn
 
+// Typesafe Resolvers
 resolvers += Resolver.typesafeRepo("releases")
 
+// S3 based SBT resolver
+resolvers ++= Seq(
+  "HAT Library Artifacts Releases" at "https://s3-eu-west-1.amazonaws.com/library-artifacts-releases.hubofallthings.com",
+  "HAT Library Artifacts Snapshots" at "https://s3-eu-west-1.amazonaws.com/library-artifacts-snapshots.hubofallthings.com",
+  "HAT Library Artifacts Releases" at "https://s3-eu-west-1.amazonaws.com/library-artifacts-releases.hubofallthings.com"
```
Comment (augustinas) [inline project/plugins.sbt:10]: Releases resolver is duplicated between lines 8 and 10. This line can be safely removed.

File: project/plugins.sbt [hunk 3]
```diff
@@ -17,14 +26,13 @@ addSbtPlugin("com.typesafe.sbt" % "sbt-digest" % "1.1.4")
...
 addSbtPlugin("org.hatdex" % "sbt-slick-postgres-generator" % "0.0.11")
 
 // run "sbt dependencyUpdates" to check maven for updates or "sbt ";dependencyUpdates; reload plugins; dependencyUpdates" for sbt plugins
 addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.3.4")
 
 // ScalaFMT
-addSbtPlugin("org.scalameta" %% "sbt-scalafmt" % "2.4.0")
+addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.3.4")
+
+addSbtPlugin("io.dataswift" % "sbt-scalafmt-common" % "0.1.1-SNAPSHOT")
```
Comment (augustinas) [inline project/plugins.sbt:38]: Does scala fmt has a rule for auto-inserting new line at EOF?

Comment (dataswifty) [reply project/plugins.sbt:38]: It's on by default, and in fact, you can't turn it off.

Go UNIX!

**Pull request merged**